Resources/Prototypes/Entities/Clothing/Back/backpacks.yml

```diff
@@ -42,7 +42,7 @@
 - type: entity
   parent: ClothingBackpack
   id: ClothingBackpackEngineering
-  name: engineering backpack
+  name: industrial backpack
```

Comment:
Is that what it's called in ss13?

Reply:
Yep pulled that from TG IIRC because it's used for both mining and all engineering roles not just engineer. (Only fireproof backpack too)

Reply:
We don't have mining and at this stage will never have it